### DISCUSSION TITLE
Cart block and Mix'n Match container

### DISCUSSION THREAD

**Discussion by louis2b** [woocommerce/woocommerce#45090] [WooCommerce, Blocks, and Site Editing]
Hi there, I have tested the blocks and would be happy to see this coming live and compatible with more extensions. I provide feedback regarding compatibility with Mix n Match although I am not sure it would be relevant as it is not yet planned. 

Purchasing a Mix and Match container especially when the price is fixed and not per item based, the classic behavior of cart is to block the ability to change quantity in the cart (for the items inside the mix and match products). If one purchases a box with 12 items and a fixed price, it cannot be changed to 20 items in the cart. 

With blocks, the items quantity selectors can be used for all elements of the cart, so that rule is no longer respected and a customer can change the number of items in the cart without affecting the price. 

I hope that helps, thanks for this great upcoming feature

**Comment by opr** [comment]
Hi there @louis2b  👋🏼 

Thanks for taking the time to make us aware that Mix and Match products isn't working with the Checkout and Cart blocks. These blocks are still in early access (via the feature plugin) and it's expected that some existing Woo Extensions that integrate with the current checkout flow in WooCommerce core will not work with the blocks.

We're currently working on adding extensibility for the blocks so extensions have a way to make sure they integrate with the new flows we're developing. Once that is done we'll have docs prepared and will be reaching out to Woo Extension developers to help them with getting their extensions integrated.

Until then you can keep on top of the [status of the Cart & Checkout Blocks in this documentation](https://docs.woocommerce.com/document/cart-checkout-blocks-support-status/)

**Comment by opr** [comment]
We make note of all feedback we receive and we also close issues with the feedback as active issues are used for bugs or feature/enhancements. It does not mean that we don't note or review the feedback. Closed issues can also still be commented on.

**Comment by helgatheviking** [comment]
FWIW, in Mix and Match 2.0 there is now preliminary compatibility between MNM and cart blocks. However, it could be improved by [making it easier to show the child item quantities in a disabled quantity input](https://github.com/woocommerce/woocommerce-blocks/issues/6644) instead of via a workaround.
